### ISSUE TITLE
Create addon version

### ISSUE THREAD

**Issue by lstor** [jonimake/arma-ccip#5]
Convert script into addon version that will automatically run whenever you get into a supported plane.

**Comment by lstor** [comment]
From a two-minute glance at the code, it seems to me that a few changes at the end of `jonimake_ccip.sqf` could do the trick. Currently, the script assumes that it is being run on a :airplane: . However, if tweaked into waiting for the player to get into a plane instead, it could work. What do you think, @jonimake?

**Comment by jonimake** [comment]
Everything is fine by my as long as it isn't busy-waiting ;)
I'm kinda swamped with real life stuff though, pull requests are welcome :)

I'll try to look into that though. I did try searching for some tutorials regarding addon configuration and packing, but I was unable to do it sadly. I don't want to use the CBA extended event handlers either, I want the script to be completely free of external dependencies.

**Comment by lstor** [comment]
Yeah, I'm hoping to find the time to look into it myself and send a pull request :) CCIP is the single most essential feature of Arma that is missing from the vanilla game, I think.

As soon as I have the time to look at it, I'll try to find a solution that doesn't require CBA/XEH or anything else, and that doesn't chew up too much CPU.

**Comment by eRazeri** [comment]
This as an AGM Module would be a pretty good idea:
https://github.com/KoffeinFlummi/AGM

**Comment by lstor** [comment]
Since @jonimake has released `arma-ccip` under a reasonable licence, the MIT licence, the AGM boys (@KoffeinFlummi and @commy2) are free to convert it into an addon and/on redistribute it with AGM, as long as the licence text and attribution is included along with the module.

Kudos to @jonimake for being an example to the community in selecting a licence :+1:
